app-emacs/calfw/calfw-1.6-r1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
```

Comment:
since we already rev-bump, let's also bump EAPI to 8

Reply:
Sure thing! Will do so in next commit